File: app/src/main/java/audiolibros/example/com/audiolibros/fragments/PreferenciasFragment.java
package audiolibros.example.com.audiolibros.fragments;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import audiolibros.example.com.audiolibros.R;

/**
 * Created by Miguel Á. Núñez on 15/02/2018.
 */

public class PreferenciasFragment extends PreferenceFragment {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: app/src/main/java/audiolibros/example/com/audiolibros/fragments/PreferenciasFragment.kt
package audiolibros.example.com.audiolibros.fragments

import android.os.Bundle
import android.preference.PreferenceFragment

import audiolibros.example.com.audiolibros.R

/**
 * Created by Miguel Á. Núñez on 15/02/2018.
 */

class PreferenciasFragment : PreferenceFragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferences)
    }
}
